test/Makefile

```diff
@@ -6,12 +6,25 @@ generate-mocks:
 
 .PHONY: test-e2e-fork12-pessimistic
 test-e2e-fork12-pessimistic: stop
-	./run-e2e.sh fork12 pessimistic
-	export BATS_TEST_TIMEOUT=1800
-	bats bats/pp/bridge-e2e.bats \
+	L2_ETH_RPC_URL=$$( { ./run-e2e.sh fork12 pessimistic cdk-erigon-rpc-001 2>&1 | tee /dev/stderr; } | tail -n1 ) && \
+	echo "L2_ETH_RPC_URL=$$L2_ETH_RPC_URL" && \
```

Comment:
I guess this was added for debugging purposes. Maybe remove this when we get it working?